.github/workflows/push-dockerhub-readme.yml

```
@@ -0,0 +1,23 @@
+name: push readme to dockerhub
+
+on:
+  push:
+    paths:
+      - Docker/README.md
+    branches:
+      - latest
```

```suggestion
      - edge
```

Branch would be `edge` though